### ISSUE TITLE
Issue No. 1

### ISSUE THREAD

**Issue by rimantas-not** [rimantas-not/Horizon-But-DIY#1]
I would call it, oh man, i wish i did that before printing. basically, mainly i do stuff with some consideration on critical measurements. This  bites sometimes.

**Comment by willdmo** [comment]
This is in agreement with my most designs. I measure, think, draw and try out. I should do assemblies in CAD more often to get the tolerances right

**Comment by rimantas-not** [comment]
Yeah, i do that too, but sometimes, brains stops doing some work. 
Or a 3D printer, liwk today, decided to throw away usually good enpugh tolerances.
